service-app/internal/aws/job_queue.go

```diff
@@ -0,0 +1,61 @@
+package aws
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/ministryofjustice/opg-scanning/config"
+	"github.com/ministryofjustice/opg-scanning/internal/types"
+)
+
+type AwsQueue struct {
+	SqsClient *sqs.SQS
+	QueueURL  string
+}
+
+func NewAwsQueue(cfg *config.Config) (*AwsQueue, error) {
+	sess, err := session.NewSession(&aws.Config{
+		Region:   aws.String(cfg.Aws.Region),
+		Endpoint: aws.String(cfg.Aws.Endpoint),
+	})
+	if err != nil {
+		return nil, fmt.Errorf("failed to create AWS session: %w", err)
+	}
+
+	return &AwsQueue{
+		SqsClient: sqs.New(sess),
+		QueueURL:  cfg.Aws.JobsQueueURL,
+	}, nil
+}
+
+func (q *AwsQueue) QueueSetForProcessing(ctx context.Context, scannedCaseResponse *types.ScannedCaseResponse, fileName string) (MessageID *string, err error) {
+	// Create a message structure
+	message := map[string]interface{}{
```

Comment:
I think from my reading of the ticket, this will need to be of the form:

```
{
__name__: “Ddc\Job\FormJob“,
content: "php serialisation of the UID+Filename object"
}
```

in order to work with the existing Sirius queue handler. @ramchale am I correct in that reading?

Reply:
On the ddc queue processing side, it looks like it's expecting the body to be a json structure 
```
{
"filename": "a file name",
"uid": "a uid with or without seperators, as it strips them out"
}
```
So I think this is okay, but the keys need to be lowercase

Reply:
Ok well noted

Reply:
@ramchale @mattmachell this is resolved